README.md

```diff
@@ -46,17 +46,18 @@ real distro installer (e.g. one that is not tailored specifically for
 Pinebook Pro). However some problems are so ackward for users the
 following temporary hacks have been tolerated.
 
- * The ALSA state cache is pre-configured. This ensured that audio will
-   work out-of-the-box. This is temporary until upstream ALSA UCM support
-   for Pinebook Pro is added.
- * The kernel is booted with maxcpus=4 to ensure that is does not use
-   the Cortex A72 cores during initialization. This works around a
-   bootloader problem that hands these cores to the kernel with clock
-   speed set so slow that it compromises boot times. The disabled cores
-   will be re-enabled later in the boot process. Be warned that to make
-   this work some kernel hardening must be disabled (ex:
-   HARDEN_EL2_VECTORS).
- 
+* The ALSA state cache is [pre-configured](var/lib/alsa/asound.state).
+  This ensured that audio will work out-of-the-box. This is temporary
+  until upstream ALSA UCM support for Pinebook Pro is added.
+* The kernel is booted with `maxcpus=4` to ensure that is does not use
+  the Cortex A72 cores during initialization. This works around a
+  bootloader problem that hands these cores to the kernel with clock
+  speed set so slow that it compromises boot times. The disabled cores
+  will be re-enabled later in the boot process.
+
+> WARNING: To make this work some kernel hardening must be disabled (ex:
+  `HARDEN_EL2_VECTORS`).
+
```

Comment:
If we outdent this we need to make clear which of the bulletted items the warning applies to. Additionally I have a stylistic preference for italics over blockquoting for warnings.

Maybe the following:

Warning: *To make the `maxcpus=4` hack work some optional kernel hardening features must be disabled (ex: [`HARDEN_EL2_VECTORS`](https://lists.cs.columbia.edu/pipermail/kvmarm/2018-March/030321.html))*

~~~
Warning: *To make the `maxcpus=4` hack work some optional kernel hardening features must be disabled (ex: [`HARDEN_EL2_VECTORS`](https://lists.cs.columbia.edu/pipermail/kvmarm/2018-March/030321.html))*
~~~

BTW we need to be careful with this warning not to make it excessively scary (which is why I did not originally outdent it). I don't think the feature we have to disable is even present in the v4.4.x vendor kernel (because it is hardening feature rather than a security fix) and there is a risk the user will incorrectly perceive the security risk of this kernel as worse than running the v4.4.x vendor kernel (which I think is probably wrong, even with the LTS patches merged into the vendor kernel).

Reply:
Thanks for your feedback! I like the proposal, it won't scare users :)